ui/src/app/utils/runtime-utils.tsx

```diff
@@ -219,22 +261,33 @@ const toRuntimeConfig = (runtime: Runtime): RuntimeConfig => {
       computeType: ComputeType.Standard,
       machine: findMachineByName(runtime.gceConfig.machineType),
       diskSize: runtime.gceConfig.diskSize,
```

Comment:
hm, we still have `diskSize` after this change. WIll that always be set?

Reply:
> Note: you will need to merge or rebase with the Angular->React - likely it just affects the diskStore initialization.

Sorry, do you mean I need to merge/rebase with the latest master since there are some codebase changes of Angular->React ?
> 
> Do you have the runtime panel spec changes?

No, there are some code format diffs with the master after I run `yarn lint --fix` ... I have to revert them manually.